keras_rs/src/losses/pairwise_hinge_loss.py

```diff
@@ -0,0 +1,34 @@
+from keras import ops
+
+from keras_rs.src import types
+from keras_rs.src.api_export import keras_rs_export
+from keras_rs.src.losses.pairwise_loss import PairwiseLoss
+from keras_rs.src.losses.pairwise_loss import pairwise_loss_subclass_doc_string
+
+
+@keras_rs_export("keras_rs.losses.PairwiseHingeLoss")
+class PairwiseHingeLoss(PairwiseLoss):
+    def pairwise_loss(self, pairwise_logits: types.Tensor) -> types.Tensor:
+        return ops.relu(ops.subtract(ops.array(1), pairwise_logits))
+
+
+PairwiseHingeLoss.__doc__ = (
+    pairwise_loss_subclass_doc_string.replace(
```

Comment:
I've seen the `replace` approach used in Keras, but I think the `format` approach is a bit cleaner: https://github.com/keras-team/keras/blob/master/keras/src/applications/mobilenet_v2.py#L499